plugin/evm/config.go

```diff
@@ -35,6 +35,7 @@ const (
 	defaultPriorityRegossipTxsPerAddress          = 16
 	defaultOfflinePruningBloomFilterSize   uint64 = 512 // Default size (MB) for the offline pruner to use
 	defaultLogLevel                               = "info"
+	defaultLogJSONFormat                          = false
```

Comment:
```suggestion
	defaultLogJSONFormat                          = true
```

Reply:
coreth/avalanchego logs are not json by default. making this json by default would seem a little weird among others.

Reply:
Ah ok, I misunderstood something @StephenButtolph mentioned and thought there was a request to switch everything to use JSON logs by default.